aicsimageio/readers/tiff_reader.py

```diff
@@ -132,6 +135,31 @@ def scenes(self) -> Tuple[str, ...]:
 
         return self._scenes
 
+    @property
+    def physical_pixel_sizes(self) -> PhysicalPixelSizes:
+        """Return the physical pixel sizes of the image."""
+        if self._physical_pixel_sizes is None:
+            with self._fs.open(self._path) as open_resource:
+                with TiffFile(open_resource) as tiff:
+                    tags = tiff.series[self.current_scene_index].pages[0].tags
+
+            if tiff.is_imagej:
+                unit = tiff.imagej_metadata["unit"]
+                z_size = tiff.imagej_metadata.get("spacing", None)
+            else:
+                unit = tags["ResolutionUnit"].value
+                z_size = None
+
+            scalar = _NAME_TO_MICRONS.get(unit, 1)
+            dx, nx = tags["XResolution"].value
```

Comment:
the value of the tag is a tuple?  Just for my understanding, what are dx and nx?

Reply:
yeah, it's a 2-tuple.  they stand (quite uselessly) for "denominator" and "numerator" :joy:

let me look that up and provide a proper comment.  I have a vague recollection of these resolution tags somehow being in inverse space (like number of pixels per unit area, rather than unit are per pixel), but let's make sure the code is clearer :)

Reply:
It would be great to include whatever you find in a comment in the code!